External/CoordGen/CoordGen.h

```diff
@@ -132,6 +172,20 @@ unsigned int addCoords(T& mol, const CoordGenParams* params = nullptr) {
         BOOST_LOG(rdWarningLog) << "unrecognized bond type";
     }
     bnds[obnd->getIdx()] = bnd;
+    if (params->minimizeOnly) {
+      bondLenAccum += (mol.getConformer().getAtomPos(obnd->getBeginAtomIdx()) -
+                       mol.getConformer().getAtomPos(obnd->getEndAtomIdx()))
+                          .length();
+    }
+  }
+  double avgBondLen = 1.0;
+  if (params->minimizeOnly) {
+    if (nSingle) {
+      avgBondLen = singleBondLenAccum / nSingle;
```

Comment:
Will this value every be used in practice?  I'm assuming mol.getNumBonds() is always > 1 and will override this value below.

Reply:
right, there is a missing "else" here